d3a_api_client/commands.py

```diff
@@ -0,0 +1,72 @@
+from d3a_api_client.enums import Commands, command_enum_to_command_name
+
+
+class ClientCommandBuffer:
+
+    def __init__(self, ):
+        self._commands_buffer = []
+
+    def offer_energy(self, area_uuid, energy, price):
+        area_uuid = area_uuid
+        args = {energy, price}
+        action = Commands.OFFER
+        self.execute(area_uuid, action, args)
```

Comment:
The 3 separate variables are redundant here. The method will be easy to read even if you use the arguments directly: `self.execute(area_uuid, Commands.OFFER, {energy, price})`.